hc/front/views.py

```diff
@@ -206,6 +207,59 @@ def update_nag_interval(request, code):
     return redirect("hc-checks")
 
 
+@login_required
+def shopify_alerts(request, code):
+    assert request.method == "POST"
+
+    return redirect("hc-checks")
+
+
+@login_required
+def create_shopify_alerts(request):
```

Comment:
This is a very long method. In future consider breaking it down into different methods